src/projects-card/ProjectsCard.css

```diff
@@ -0,0 +1,21 @@
+.mantine-Card-root {
+    flex-shrink: 0;
+    box-shadow: 0px 4px 4px 0px rgba(0, 0, 0, 0.25);
+    border-radius: 30px;
+}
+
+.mantine-Card-section {
+    margin: 0;
+}
```

Comment:
In Figma it looks like theres more padding around the text part of the card, especially bottom. Could adjust padding here to match!